Properly handle pgp keychains generated by Scala CLI (#1987)
VirtusLab/scala-cli: website/docs/commands/publishing/publish-setup.md

Generate a PGP key pair for publishing with
```bash
scala-cli --power config --create-pgp-key --pgp-password MY_CHOSEN_PASSWORD
```

It's not mandatory, although recomended, to use a password to encrypt your keychains.

> To store the private keychain in an unencrypted form use `--pgp-password none`, use
> `--pgp-password random` for Scala CLI to randomly generate a password for you.

Reviewer: NIT
```suggestion
To store the private keychain in an unencrypted form use `--pgp-password none` 
To randomly generate a pasword, use `--pgp-password random` instead.
```
